packages/ui-heading/src/Heading/README.md

```diff
@@ -4,6 +4,31 @@ describes: Heading
 
 Heading is a component for creating typographic headings.
 
+## Variant
+
+Variant takes care of - almost - all use-cases when it comes hedings on pages. Their name reflects the places they meant to be used. It sets the `level` prop and takes care of the style of the heading
+We recommend using `variants` instead of the level (and `as`) props.
```

Comment:
Are these deprecated? If yes move their examples to the bottom of this page please

Reply:
I don't think so. I'd consider deprecating them, but this needs more investigation imho.